packages/nimble-components/src/anchor-button/tests/anchor-button.mdx

```diff
@@ -0,0 +1,26 @@
+import {
+    Controls,
+    DocsStory,
+    Meta,
+    Stories,
+    Title,
+    Description
+} from '@storybook/blocks';
+import * as anchorButtonStories from './anchor-button.stories';
+
+<Meta of={anchorButtonStories} />
+
+<Title of={anchorButtonStories} />
+<Description of={anchorButtonStories} />
+
+<DocsStory of={anchorButtonStories.outlineAnchorButton} expanded={false} />
+<Controls of={anchorButtonStories.outlineAnchorButton} />
+<Stories of={anchorButtonStories} />
```

Comment:
Everytime I change the theme I'm noticing the list of stories grow longer on the doc page. All the stories seem to get re-added to the page. Seems like a bug to report to storybook.

I'm also skeptical of the value of placing this list of stories on the docs page. Seems like the DocsStory and the Controls give the right value. I'd opt for reporting the issue to storybook to be good citizens and removing the Stories from our doc pages. @jattasNI @msmithNI 

![image](https://github.com/ni/nimble/assets/1588923/6b3b5fbc-e5b9-470c-a62e-1706f725067d)

Reply:
Good catch! I'm ok with removing the stories; they also take up a lot of vertical space so make it less likely that readers will make it to the bottom of the page to see the usage guidance, which is likely more important.

FYI @fredvisser since you're also thinking about reorganizing docs pages.

Reply:
Removed `<Stories />` from the mdx files